src/views/List.jsx

```diff
@@ -23,26 +25,41 @@ export function List({ data }) {
 	return (
 		<>
 			<p>
-				Hello from the <code>/list</code> page!
+				Hello from the <code>{listName}</code> page!
 			</p>
-			<form onSubmit={handleSubmit}>
-				<label htmlFor="search">Search:</label>
-				<input
-					type="text"
-					id="search"
-					name="search"
-					onChange={handleChange}
-					value={search}
-				/>
-				<button type="button" onClick={handleClear}>
-					x
-				</button>
-			</form>
-			<ul>
-				{filteredData.map((item) => (
-					<ListItem key={item.name} name={item.name} />
-				))}
-			</ul>
+			{loading ? (
+				<p>loading . . .</p>
+			) : data.length > 0 ? (
+				<>
+					<form onSubmit={handleSubmit}>
+						<label htmlFor="search">Search:</label>
+						<input
+							type="text"
+							id="search"
+							name="search"
+							onChange={handleChange}
+							value={search}
+						/>
+						<button type="button" onClick={handleClear}>
+							x
+						</button>
+					</form>
+					<ul>
+						{filteredData.map((item) => (
+							<ListItem key={item.name} name={item.name} />
+						))}
+					</ul>
+				</>
+			) : (
+				<>
+					<p>
+						Please add to the <code>{listName}</code> list to get started.
+					</p>
+					<button>
+						<Link to="/manage-list">Add your first item</Link>
+					</button>
+				</>
```

Comment:
😅 I am not sure about this but had a thought, do we want to also add the loader to the `manageList` view/ functionality? What do you all think? @3campos @eonflower @hun-ah ?

Reply:
That's a good thought. For context, I added a loader for the shopping list view out of necessity (we had rendering issues with old data flashing on the shopping list page). I personally don't think we need a loader for the manage list view for this issue but it is something we can consider adding later on in our project (issue 14).